docs/installation.md

```diff
@@ -88,8 +102,7 @@ You can locate the resulting build artifact at `/target/wasm32-unknown-unknown/r
 You can now start the sandbox with:
 
 ```sh
-make build-installer
-cargo run -- sandbox start
+cargo run --bin jstz -- sandbox start
```

Comment:
Some of the build commands have changed since this was written 😅

I think it should be:
```sh
make build-cli-kernel
cargo run --bin jstz -- sandbox start
```